x/swaprouter/creator_test.go

```diff
@@ -22,20 +24,23 @@ func (suite *KeeperTestSuite) TestCreatePool() {
 	}, "")
 
 	tests := []struct {
-		name              string
-		creatorFundAmount sdk.Coins
-		msg               types.CreatePoolMsg
-		expectError       bool
+		name               string
+		creatorFundAmount  sdk.Coins
+		msg                types.CreatePoolMsg
+		expectedModuleType reflect.Type
+		expectError        bool
 	}{
 		{
-			name:              "first balancer pool - success",
-			creatorFundAmount: sdk.NewCoins(sdk.NewCoin(denomA, defaultInitPoolAmount.Mul(sdk.NewInt(2))), sdk.NewCoin(denomB, defaultInitPoolAmount.Mul(sdk.NewInt(2)))),
-			msg:               validBalancerPoolMsg,
+			name:               "first balancer pool - success",
+			creatorFundAmount:  sdk.NewCoins(sdk.NewCoin(denomA, defaultInitPoolAmount.Mul(sdk.NewInt(2))), sdk.NewCoin(denomB, defaultInitPoolAmount.Mul(sdk.NewInt(2)))),
+			msg:                validBalancerPoolMsg,
+			expectedModuleType: gammKeeperType,
 		},
 		{
-			name:              "second balancer pool - success",
-			creatorFundAmount: sdk.NewCoins(sdk.NewCoin(denomA, defaultInitPoolAmount.Mul(sdk.NewInt(2))), sdk.NewCoin(denomB, defaultInitPoolAmount.Mul(sdk.NewInt(2)))),
-			msg:               validBalancerPoolMsg,
+			name:               "second balancer pool - success",
+			creatorFundAmount:  sdk.NewCoins(sdk.NewCoin(denomA, defaultInitPoolAmount.Mul(sdk.NewInt(2))), sdk.NewCoin(denomB, defaultInitPoolAmount.Mul(sdk.NewInt(2)))),
+			msg:                validBalancerPoolMsg,
+			expectedModuleType: gammKeeperType,
```

Comment:
What's the significance of having first and second msg? Do they not get initialized anyways?